test-unit.sh

```diff
@@ -22,7 +20,7 @@ unit_test() {
   service=$1
   pushd services/$service
   install_deps
-  serverless testUnit --stage $stage
+  yarn run coverage
```

Comment:
Part of changes to close #263 
Instead of running yarn run coverage via serverless scripts, we invoke it directly